src/ProductList.js

```diff
@@ -1,28 +1,44 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+
+import { fetchLineItems } from './store/lineitem';
+import { getProductByPg, fetchProducts } from './store/product'
+import Pager from './Pager';
+
 import { lineItems, fetchLineItems } from './store/lineitem';
```

Comment:
Remove this line. Getting an error because you declared fetchLineItems twice.

Reply:
@5t4mp5 done.